src/lyra/core/expressions.py

```diff
@@ -138,6 +139,10 @@ def visit_LengthIdentifier(self, expr: 'LengthIdentifier'):
     def visit_ListDisplay(self, expr: 'ListDisplay'):
         """Visit of a list display."""
 
+    # @abstractmethod
+    # def visit_DictDisplay(self, expr: 'DictDisplay'):
+    #     """Visit of dictionary display."""
```

Comment:
This is commented out just so you do not have to update all subclasses, right? If so, that is fine ;)